web_widget_float_formula/README.rst

```diff
@@ -1,55 +1,88 @@
+   :alt: License: AGPL-3
 
-* Possibility to tip a text like "=45 + 4/3 - 5 * (2 +1)";
-* if the formula is correct, The result will be computed and displayed;
-* if the formula is not correct, the initial text is displayed;
+========================
+Formulas in Float Fields
+========================
 
-Technical informations
-----------------------
+This module allows the use of simple math formulas in integer/float fields 
+(e.g. "=45 + 4/3 - 5 * (2 + 1)").
 
-* Overloads "instance.web.form.FieldFloat"; (so works for fields.integer &
-  fields.float);
-* To compute, the module simply use the eval() javascript function;
-* Rounding computation is not done by this module (The module has the same
-  behaviour if the user tips "=1/3" or if he tips "0.33[...]");
-* avoid code injonction by regexpr test: "=alert('security')" is not valid;
+* Only supports parentheses, decimal points, thousands separators, and the 
+  operators "+", "-", "*", and "/"
+* Will use the decimal point and thousands separator characters associated 
+  with your language
+* If the formula is valid, the result will be computed and displayed, and the 
+  formula will be stored for editing
+* If the formula is not valid, it's retained in the field as text
+
+**Technical Details**
+
+* Overloads web.form_widgets.FieldFloat (so it works for fields.integer & 
+  fields.float)
+* Uses the eval() JS function to evaluate the formula
+* Does not do any rounding (this is handled elsewhere)
+* Avoids code injection by applying strict regex to formula prior to eval() 
+  (e.g. "=alert('security')" would not get evaluated)
+
+Installation
+============
+
+To install this module, simply follow the standard install process.
+
+Configuration
+=============
+
+No configuration is needed or possible.
 
 Usage
 =====
 
-See demo here Video: http://www.youtube.com/watch?v=jQGdD34WYrA&hd=1
+Install and enjoy. A short demo video can be found at 
+http://www.youtube.com/watch?v=jQGdD34WYrA.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/162/9.0
 
-Roadmap / Limit
-===============
-* Only supports the four operators: "+" "-" "*" "/" and parenthesis;
+Known Issues / Roadmap
+======================
 
 Bug Tracker
 ===========
 
 Bugs are tracked on `GitHub Issues <https://github.com/OCA/web/issues>`_.
 In case of trouble, please check there if your issue has already been reported.
-If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
-`here <https://github.com/OCA/web/issues/new?body=module:%20web_widget_float_formula%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+If you spotted it first, help us smash it by providing detailed and welcomed 
+feedback.
 
 Credits
 =======
 
+Images
+------
+
+* Odoo Community Association: 
```

Comment:
The module does have an `icon.png`. It's just not showing up in the PR because I didn't change it in any way. Do you still want me to remove this section?